web/book/tests/documentation/snapshots/documentation__book__language-features__regex__regex-expressions__3.snap

```diff
@@ -7,5 +7,5 @@ SELECT
 FROM
   tracks
 WHERE
-  name ~ '\(I Can''t Help\) Falling'
+  (name) ~ ('\(I Can''t Help\) Falling')
```

Comment:
These changes are intentional?

Reply:
No, I saw these too — I'm not sure why they happen...

Reply:
Ah sorry!

(I blame the comment syntax :D )